CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ git logs & PR history.
 
 - Raise `ArgumentError` when passing `role = nil` to `only_host_with_role()` or `find_at_most_one_host_with_role()`
 - Use `install_package_with_rpm` in `add_el_extras`
+- Exit early on --help/--version/--parse-only arguments instead of partial dry-run
```

Comment:
looks like this line has been pushed down into 3.36.0 with the new release, so this will have to be put back up into the Unreleased section